hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamerTestBase.java

```diff
@@ -197,23 +193,22 @@ protected static void writeCommonPropsToFile(FileSystem dfs, String dfsBasePath)
     UtilitiesTestBase.Helpers.savePropsToDFS(props, dfs, dfsBasePath + "/" + PROPS_FILENAME_TEST_SOURCE);
   }
 
-  @BeforeEach
-  public void setup() throws Exception {
-    super.setup();
-    TestDataSource.returnEmptyBatch = false;
+  @BeforeAll
+  public static void initClass() throws Exception {
+    UtilitiesTestBase.initTestServices(false, true, false);
```

Comment:
zookeeper service is not needed (3rd arg). the test with kafka is done with embedded zookeeper from spark's kafka test utils